cypress-tests/cypress/fixtures/confirm-body.json

```diff
@@ -27,7 +27,7 @@
     "address": {
       "state": "New York",
       "city": "New York",
-      "country": "US",
+      "country": "NL",
```

Comment:
why do we need this change? ig this country has been used by the card pm test cases. please revert this